CHANGELOG.md

```diff
@@ -26,11 +26,18 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.6.0...master
 
+### Added
+- Add `ActionReport` interface which is necessary for [JPERF-537].
+- Add `reports` property to `Verdict`. Resolve [JPERF-537].
+
+### Deprecated
+- Deprecate `failedActions` property in `Verdict`.
```

Comment:
This shows that we're putting experimental APIs publicly. The safer and quicker way would be to experiment locally and promote to API only when it's battle-tested.
Coming back to [my previous comment](https://ecosystem.atlassian.net/browse/JPERF-537?focusedCommentId=430040&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-430040), perhaps it's best to actually copy-paste the non-parametric judges, figure out the action rerunning and then try to put it here.

Reply:
Agree. Another alternative is to still remain on version 0